### ISSUE TITLE
Remove PDFs from Site Map

### ISSUE THREAD

**Issue by nutjob4life** [EDRN/P5#107]
Kara Smigel, Gary Morin, and others on the accessibility team report that the Site Map at https://edrn.jpl.nasa.gov/portal/sitemap has ~18 PDF files that should be replaced.

**Comment by nutjob4life** [comment]
Removed "File" from "Navigation" settings as "included content types".
